specification/communication/data-plane/Microsoft.CommunicationServicesIdentity/preview/2021-03-31-preview/CommunicationIdentity.json

```diff
@@ -0,0 +1,118 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "CommunicationIdentityClient",
+    "description": "Azure Communication Identity Service",
+    "version": "2021-03-31-preview"
+  },
+  "paths": {
```

Comment:
Let's make sure all the routes and not just the new ones show up or the SDK cannot support the other routes

Reply:
So you want to add all endpoints into this new API? I thought there will be new SDK for this one and it can be taken as 2 separated things.

Reply:
All the routes should show up here so the new version of sdk would be based on this version of the swagger.

Reply:
Added all routes

Reply:
Also TURN endpoint?

Reply:
After discussion in team we removed Turn token from our swagger.